OSOE-1057: Certain UI tests started to fail in NuGetTest

Piedone opened the pull request:
[OSOE-1057](https://lombiq.atlassian.net/browse/OSOE-1057)
Fixes #991

[OSOE-1057]: https://lombiq.atlassian.net/browse/OSOE-1057?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Piedone committed:
Submodule src/Utilities/Lombiq.NodeJs.Extensions updated

Submodule src/Utilities/Lombiq.NodeJs.Extensions:
    > 13faeda - Updating sample to reference the latest NuGet

Piedone committed:
Submodule test/Lombiq.UITestingToolbox updated

Submodule test/Lombiq.UITestingToolbox:
    > ba35feb6 - Disabling Chromium notification prompts
    > 75885d16 - Making FillEmailTestFormAsync more reliable
    > 5b657a6a - Making navigation more reliable
    > 42ce7282 - Removing unnecessary exception usage
    > 726dc260 - More specific StaleElementReferenceException retries for clicks instead of in ExecuteSection methods

Piedone committed:
Temporarily disabling NuGetTest build

Piedone committed:
Submodule test/Lombiq.UITestingToolbox updated

Submodule test/Lombiq.UITestingToolbox:
    > ad546f0f - Revert "Making navigation more reliable"

Piedone committed:
Temp workflow config

Piedone committed:
Submodules src/Modules/Lombiq.DataTables, test/Lombiq.UITestingToolbox updated

Submodule src/Modules/Lombiq.DataTables:
    > 7117e7e - Using safer element retrieval methods in UI test

Submodule test/Lombiq.UITestingToolbox:
    > 1469d88d - Leftover
    > 1fe97178 - Adding further *WithStaleRetries methods
    > 2447b37f - Adding stale-retrying Get* methods
    > 9f40fec4 - Better docs

Piedone committed:
Reverting workflow

Piedone committed:
Updating Lombiq NuGet references

Piedone committed:
Code styling

Piedone committed:
Updating Lombiq NuGet references

Piedone committed:
Temporary cancel-workflow-on-failure: ‘false'

Piedone committed:
Temporary cancel-in-progress-for-this-pr: 'false'

Piedone committed:
Fixing UITT NuGet version

Piedone committed:
Fixing NuGetTest tests

Piedone committed:
Removing unused htmlvalidate config file

Piedone committed:
Reverting workflow

BenedekFarkas committed:
Code styling

BenedekFarkas commented:
LGTM code-wise, but some tests still seem flaky...

`TrumbowygBlogPostsFeatureShouldWork` (NuGet) failed on your last commit, which I tested locally:

- Failed twice in the code solution.
- Than it was successful in the code solution while debugging, then also in normal execution.
- Successful in the NuGet solution for the first try.

Then I pushed a code styling change, where `TrumbowygBlogPostsFeatureShouldWork` was successful, but `ConditionallyEnabledFeaturesShouldWorkCorrectlyOnTenants` failed with [the same error as on my issue branch](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/13822913564?pr=985).

Piedone committed:
UI test retry interval set at 30s in GHA

Piedone committed:
Temporary workflows

Piedone committed:
Submodule test/Lombiq.UITestingToolbox updated

Submodule test/Lombiq.UITestingToolbox:
    > 9ea22aa1 - Making notification message-checking and clicking methods safer, refactoring reliability methods
    > e0563428 - Logging for test retry interval

Piedone committed:
Fixing build error

Piedone committed:
Submodule tools/Lombiq.GitHub.Actions updated

Submodule tools/Lombiq.GitHub.Actions:
    > 68185147 - Adding missing type

Piedone committed:
Submodule test/Lombiq.UITestingToolbox updated

Submodule test/Lombiq.UITestingToolbox:
    > 9ea22aa1 - Making notification message-checking and clicking methods safer, refactoring reliability methods
    > e0563428 - Logging for test retry interval

Piedone committed:
Temporarily running root solution builds on GitHub-hosted runners too

Piedone committed:
Submodule src/Modules/Lombiq.Hosting.Tenants updated

Submodule src/Modules/Lombiq.Hosting.Tenants:
    > 98977d0 - Fixing TestConditionallyEnabledFeaturesAsync flakyness
    > d02c212 - Update Readme.md

Piedone committed:
Temporarily not running NuGetTest builds

Piedone committed:
Submodules src/Modules/Lombiq.DataTables, src/Modules/Lombiq.Hosting.Tenants, test/Lombiq.UITestingToolbox updated

Submodule src/Modules/Lombiq.DataTables:
    > 7fc76ce - Adapting to UITT API change

Submodule src/Modules/Lombiq.Hosting.Tenants:
    > c63f570 - Adapting to UITT API change

Submodule test/Lombiq.UITestingToolbox:
    > 53d8271b - Now actually all EementRetrievalUITestContextExtensions methods retry on StaleElementReferenceException
    > 18b25838 - Missing change
    > afa10f59 - All ElementRetrievalUITestContextExtensions methods now retry on StaleElementReferenceException

Piedone committed:
Submodule test/Lombiq.UITestingToolbox updated

Submodule test/Lombiq.UITestingToolbox:
    > 20672a62 - Skipping interactive JavaScriptTests.Sandbox

Piedone committed:
Nothing actually changed in Lombiq.Hosting.Tenants

Piedone committed:
Making page navigation methods reliable too

Piedone committed:
Dummy change to kick off build

Piedone committed:
Revert "Dummy change to kick off build"

This reverts commit 2e11b61185e5d1b8651402b7d18d1419e1c2a485.

Piedone committed:
Submodule test/Lombiq.UITestingToolbox:

    > 121c9d25 - Updating VerifyNavbar Windows Chrome images
    > 0faafe6f - Making VerifyNavbar's code match the documentation
    > 75e99fda - Removing useless but potentially failing TestAdminBackgroundTasksAsMonkeyRecursivelyShouldWorkWithAdminUser sample

Piedone committed:
Updating Lombiq NuGet references

Piedone committed:
Revert "Temporarily not running NuGetTest builds"

This reverts commit ef52bdcf9677354b98b4f0820ea1a0897f201aa1.

Piedone committed:
Pointing GHA to issue branch

Piedone committed:
Back to full workflows

Piedone committed:
Submodules src/Modules/Lombiq.DataTables, src/Modules/Lombiq.EmailClient, test/Lombiq.UITestingToolbox updated

Submodule src/Modules/Lombiq.DataTables:
    > 093d04a - Reverting unnecessary UITT reference change

Submodule src/Modules/Lombiq.EmailClient:
    > 1197dff - Updating Lombiq NuGet references

Submodule test/Lombiq.UITestingToolbox:
    > aafa56bc - Code styling
    > 02c11a3a - Catching "Node with given id does not belong to the document" exceptions too when checking for navigation

DemeSzabolcs committed:
Trying to run test 10 times

DemeSzabolcs committed:
Revert "Trying to run test 10 times"

This reverts commit b8994c0cf4b5f5280d55ae99d1456ca08168c0ef.

DemeSzabolcs committed:
Using dev branch in workflows

DemeSzabolcs committed:
Updating submodules to latest dev
